src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
```

Comment:
The "development" tab is still showing up, but is empty (as expected, since these docs are about CSS). Is there a way to hide that tab?

Reply:
Unfortunately there isn't a way right now, but I brought up to Steve and opened an issue on the repo. Issue link below. 
https://github.com/blackbaud/skyux-docs-tools/issues/64

Reply:
I think the Development tab should probably be here regardless, shouldn't it? I mean, shouldn't we have the Installation section on the Development tab regardless of whether we have any development properties or code samples? I think that if we add the missing `moduleName` and `moduleSourceCodePath` attributes to the `sky-docs-demo-page element, then that Installation section will automatically show up on the Development tab.

Reply:
Got it. Thanks @Blackbaud-DeniseCisneros !

@blackbaud-johnly yeah, you're totally right - we'll need install information for the module. Thanks for pointing that out.

Reply:
@Blackbaud-AlexKingman, given that we need the installation information, does that mean that we should just close the docs-tools issue that Denise filed? I mean, is it likely that there will ever be a scenario where a component doesn't at least require installation information?

Reply:
I'm not sure yet? It seems unlikely, but I don't know for sure without going through each repo.

Reply:
Resolving this conversation so that we can move forward with this PR. We can circle back to these docs as necessary.